Dizzy/Screens/Login/AppInfosView.swift

```diff
@@ -0,0 +1,199 @@
+//
+//  AppInfoView.swift
+//  Dizzy
+//
+//  Created by stas berkman on 05/06/2019.
+//  Copyright © 2019 Dizzy. All rights reserved.
+//
+
+import UIKit
+
+protocol AppInfosViewDelegate: class {
+    func appInfoButtonPressed(appInfosView: AppInfosView, type: AppInfoType)
+}
+
+enum AppInfoType: String {
+    case about = "About"
+    case termsOfUse = "Terms of Use"
+    case privacyPolicy = "Privacy Policy"
+    case contactUs = "Contact Us"
+}
+
+class AppInfoView: UIView {
+    let stackView = UIStackView()
+    let infoButton = UIButton()
+    let separatorView = UIView()
+    
+    let separatorMultipliedWidth: CGFloat = 0.6
+    let separatorHeight: CGFloat = 0.5
+    let separatorBackgroundColor: UIColor = UIColor(red: 97/255, green: 97/255, blue: 97/255, alpha: 1.0)
+    
+    var appInfoType: AppInfoType? {
```

Comment:
why wont u get it on the init? 
this way it wont be nullable and save some code and guard let

Reply:
Done!